test/libdnf/rpm/solv/test_solv_map.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "test_solv_map.hpp"
+
+#include <cstdint>
+
+
+CPPUNIT_TEST_SUITE_REGISTRATION(SolvMapTest);
+
+
+void SolvMapTest::setUp() {
+    map1 = new libdnf::rpm::solv::SolvMap(32);
```

Comment:
Interesting part is when `libdnf::rpm::solv::SolvMap(26)`, `map1->add(30);` will pass. But this is not an issue.

Reply:
Yes, this is expected; I'll also write a test + a comment that will explain this behavior.